<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mytypon.qrc">
    <normaloff>:/pictures/gear.png</normaloff>:/pictures/gear.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="uniformRowHeights">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>2</string>
         </property>
        </column>
        <item>
         <property name="text">
          <string>General</string>
         </property>
         <property name="icon">
          <iconset resource="mytypon.qrc">
           <normaloff>:/pictures/gear.png</normaloff>:/pictures/gear.png</iconset>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEnabled</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Current Typon</string>
         </property>
         <property name="icon">
          <iconset resource="mytypon.qrc">
           <normaloff>:/pictures/page_gear.png</normaloff>:/pictures/page_gear.png</iconset>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEnabled</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Grid</string>
         </property>
         <property name="icon">
          <iconset resource="mytypon.qrc">
           <normaloff>:/pictures/table.png</normaloff>:/pictures/table.png</iconset>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsEnabled</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Components libraries</string>
         </property>
         <property name="icon">
          <iconset resource="mytypon.qrc">
           <normaloff>:/pictures/chip2.png</normaloff>:/pictures/chip2.png</iconset>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Layers</string>
         </property>
         <property name="icon">
          <iconset resource="mytypon.qrc">
           <normaloff>:/pictures/layers_map.png</normaloff>:/pictures/layers_map.png</iconset>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </item>
       </widget>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>10</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>0</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>4</number>
        </property>
        <widget class="GeneralOptionsWidget" name="generalOptionsPage"/>
        <widget class="TyponOptionsWidget" name="currentTyponOptionsPage"/>
        <widget class="GridWidget" name="gridOptionsPage"/>
        <widget class="LibrariesWidget" name="librariesPage"/>
        <widget class="LayersOptionWidget" name="layersPage"/>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GeneralOptionsWidget</class>
   <extends>QWidget</extends>
   <header>generaloptionswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GridWidget</class>
   <extends>QWidget</extends>
   <header>gridwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TyponOptionsWidget</class>
   <extends>QWidget</extends>
   <header>typonoptionswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LibrariesWidget</class>
   <extends>QWidget</extends>
   <header>librarieswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LayersOptionWidget</class>
   <extends>QWidget</extends>
   <header>layersoptionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mytypon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
